libbeat/beat/beat.go

```diff
@@ -238,6 +241,8 @@ func (b *Beat) configure() error {
 		return fmt.Errorf("error loading config file: %v", err)
 	}
 
+	// init paths config:
```

Comment:
line can be removed. comment left over from original failed (more complicated) approach.